front/pages/w/[wId]/a/[aId]/execute/index.tsx

```diff
@@ -539,20 +539,27 @@ export default function ExecuteView({
     }, 0);
   };
 
-  const handleKeyPress = (event: KeyboardEvent) => {
+  const documentHandleKeyPress = (event: KeyboardEvent) => {
+    if (event.metaKey === true && event.key === "Enter" && canRun()) {
+      handleRun();
+      return;
+    }
+  };
+
+  const reactHandleKeyPress = (event: React.KeyboardEvent) => {
     if (event.metaKey === true && event.key === "Enter" && canRun()) {
       handleRun();
-      return false;
+      return;
     }
   };
 
   useEffect(() => {
-    document.addEventListener("keydown", handleKeyPress);
+    document.addEventListener("keydown", documentHandleKeyPress);
```

Comment:
The same function was used for keyPress on a DOM element and at the document level.
But they do not take the same objects as parameters, they just happened to be compatible in our use case.
So I had to create two versions (`reactHandleKeyPress` and `documentHandleKeyPress`).

Reply:
You can surely have a `X | Y` that's super sad to have two functions that repeat the same code